openapi.yaml

```diff
@@ -16985,6 +17032,346 @@ paths:
+      responses:
+        '200':
+          description: The updated placement group.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PlacementGroup'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Content-Type: application/json" \
+              -H "Authorization: Bearer $TOKEN" \
+              -X PUT -d '{
+                "label": "PG_Miami_failover_update"
+              }' \
+              https://api.linode.com/v4/placement/groups/528
+      - lang: CLI
+        source: >
+          linode-cli placement update 528 \
+            --label PG_Miami_failover_update
+    delete:
+      x-linode-grant: read_write
+      tags:
+      - Placement Groups
+      summary: Placement Group Delete
+      description: |
+        Deletes a placement group you have permission to `read_write`.
+
+        * Deleting a placement group can't be undone.
+        * All compute instances need to be [removed](#placement-group-unassign) before you can delete a placement group.
+        * If your placement group is non-compliant, you can't delete it. You need to wait for our help to bring it [compliant](/docs/products/compute/compute-instances/guides/placement-groups/#non-compliance-precedence).
+      operationId: deletePlacementGroup
+      x-linode-cli-action:
+        - delete
+        - rm
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - linodes:read_write
+      responses:
+        '200':
+          description: Delete successful
+          content:
+            application/json:
+              schema:
+                type: object
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+              -X DELETE \
+              https://api.linode.com/v4/placement/groups/528
+      - lang: CLI
+        source: >
+          linode-cli placement delete 528
+  /placement/groups/{groupId}/assign:
+    x-linode-cli-command: placement
+    parameters:
+    - name: groupId
+      in: path
+      description: ID of the placement group to look up. Run the [Placement Groups List](#placement-groups-list) operation and store the `id` for the applicable placement group.
+      required: true
+      schema:
+        type: integer
+    post:
+      x-linode-grant: read-write
+      tags:
+      - Placement Groups
+      summary: Placement Group Assign
+      description: |
+        Add one or more compute instances to your placement group. Check out our [example API workflow](/docs/products/compute/compute-instances/guides/placement-groups/#create-a-placement-group) to create a placement group and add compute instances.
+      operationId: placementGroupAddLinode
+      x-linode-cli-action: assignLinode
```

Comment:
Just to keep these commands consistent with the rest of the, could this this become `assign-linode`?

Reply:
As above.